specification/hugr.md

```diff
@@ -127,14 +127,39 @@ as *control* and the second as *target*.
 In this case, output 0 of the H operation is connected to input 0 of the
 CNOT.
 
-### Edges
-
-A `SimpleType` is the type of a value that can be sent down a wire. We
-distinguish between `ClassicType` and `LinearType`. For more details see the
-[Type System](#type-system) section.
+### Edges, ports and signatures
 
 The edges of a HUGR encode relationships between nodes; there are several *kinds*
-of edge for different relationships, and some edges have types:
+of edge for different relationships.
+
+- `Order` edges are plain directed edges, and express requirements on the ordering.
+- `Value` edges carry typed data at runtime. They have a _port_ at each end, associated
+  with the source and target nodes.
+- `Static` edges are similar to `Value` edges but carry static data (knowable at
+  compilation time).
+- `ControlFlow` edges represent possible flows of control from one part of the
+  program to another.
+- `Hierarchy` edges express the relationship between container nodes and their
+  children.
+
+`Value` and `Static` edges are sometimes referred to as _dataflow_ edges.
+A `Value` edge can carry data of any `SimpleType`: either a `ClassicType`
+(ordinary classical data) or a `LinearType` (data which cannot be copied,
+including quantum data). A `Static` edge can only carry a `ClassicType`. For
+more details see the [Type System](#type-system) section.
+
+A `Value` edge can carry any `SimpleType`; a `Static` edge can only carry a
```

Comment:
Ok, you can remove the first sentence here now (it repeats the previous paragraph almost exactly)